### PR TITLE
[Feature] add SMPLifyx support in process_smc tool

### PR DESCRIPTION
**Pull request by LazyBusyYang**
1. add SMPLifyx support in process_smc tool, smplx_data will be dumped and visualized
2. update installation.md, add mmdeploy installation guide and update torch version to 1.12.0

### CODE REVIEW

File: docs/en/installation.md [hunk 1]
```diff
@@ -38,19 +39,43 @@ source activate xrmocap
 conda install -y ffmpeg
 
 # install pytorch
-conda install -y pytorch==1.8.1 torchvision==0.9.1 cudatoolkit=10.1 -c pytorch
+conda install pytorch==1.12.0 torchvision==0.13.0 cudatoolkit=11.3 -c pytorch
```
Comment (jiaqiAA) [inline docs/en/installation.md:43]: Shall we keep the old version of the `installation.md`?

File: tools/process_smc.py [hunk 2]
```diff
@@ -117,14 +128,26 @@ def main(args):
...
-                type='SMPL',
-                gender='neutral',
-                num_betas=10,
-                keypoint_src='smpl_45',
-                keypoint_dst='smpl',
-                model_path='xrmocap_data/body_models',
-                batch_size=1)
+            if smpl_type == 'smplx':
+                body_model_cfg = dict(
+                    type='SMPLX',
```
Comment (jiaqiAA) [inline tools/process_smc.py:133]:
```suggestion
                    type=smpl_type,
```

File: docs/en/installation.md (hunk 1 above)
Comment (LazyBusyYang) [reply docs/en/installation.md:45]: Fixed in [2e69819](https://github.com/openxrlab/xrmocap/pull/71/commits/2e6981933fd4c6ca0680021655faa4820900cc15).

File: tools/process_smc.py (hunk 2 above)
Comment (LazyBusyYang) [reply tools/process_smc.py]: Fixed in [2e69819](https://github.com/openxrlab/xrmocap/pull/71/commits/2e6981933fd4c6ca0680021655faa4820900cc15).

**Review by jiaqiAA — APPROVED** [review]

**Pull request merged**